agent/util-scripts/pbench-tool-meister-start

```diff
@@ -1215,22 +1110,44 @@ if __name__ == "__main__":
         default=None,
         help="The list of system information items to be collected.",
     )
+    parser.add_argument(
+        "--orchestrate",
+        dest="orchestrate",
+        default="create",
+        choices=("create", "existing"),
+        help=(
+            "The `create` keyword directs the command to create the various"
+            " instances of the Redis server, Tool Data Sink, and Tool"
+            " Meisters, while the `existing` keyword directs the command to"
+            " use existing instances of all three."
+        ),
+    )
     parser.add_argument(
         "--redis-server",
         dest="redis_server",
         default=os.environ.get("PBENCH_REDIS_SERVER", None),
         help=(
-            "Use an existing Redis server specified by <hostname>:<port>;"
-            " implies an existing Tool Data Sink and Tool Meisters as well."
+            "Specifies the IP/port to use for the Redis server - if not"
+            " present, the defaults are used; can be present in two forms:"
+            " `<bind host>:<port>;<host>:<port>`, a semi-colon separated"
+            " IP/port specified for both how the Redis server will bind"
+            " itself, and how clients will connect; `<host>:<port>`, the"
+            " IP/port combination is used both for binding and connecting"
+            " (NOTE: binding is ignored with --orchestrate=existing)"
```

Comment:
Perhaps replace "ignored" with "not used"?  (I'll grant it's a subtle distinction, but "ignored" implies that we use something else instead....)

Ditto line 1150.